src/components/SecondaryPanes/Frames/index.js

```diff
@@ -176,6 +182,7 @@ class Frames extends Component {
     return dom.div(
       { className: "pane frames" },
       this.renderFrames(frames),
+      WhyPaused(),
```

Comment:
Removing an unnecessary connect is a great idea. I think you can go ahead with making it stateless.